UniWatch/Views/Class/Index.cshtml

```diff
@@ -0,0 +1,49 @@
+﻿@model IEnumerable<UniWatch.Models.Class>
+
+<h3>My Classes</h3>
+
+<p>
+    @Html.ActionLink("New Class", "Create", new { teacherId = (int)ViewBag.TeacherId }, new { @class = "btn btn-success" })
+</p>
+
+<div class="table-responsive">
+    <table class="table table-bordered table-striped">
+        <tr>
+            <th>
+                @Html.DisplayNameFor(model => model.Name)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.Number)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.Section)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.Year)
+            </th>
+            <th></th>
+        </tr>
+
+        @foreach (var item in Model)
```

Comment:
This doesn't work, I fixed everything else and ran the program without the view part and it works perfectly fine. When I did try your method It gave me an error. 
![image](https://cloud.githubusercontent.com/assets/14823354/20636639/28d0b4da-b336-11e6-9f3f-e5688dcbda89.png)